### DISCUSSION TITLE
AltSnap as MS Power Toys app?

### DISCUSSION THREAD

**Discussion by ItsSlimeTime** [RamonUnch/AltDrag#434] [Ideas]
There's a thread opened back in 2019 (and regularly requested since) on MS Power Toys page [https://github.com/microsoft/PowerToys/issues/269](url) requesting AltSnap / AltDrag be added as a feature.

Most of us who rely on this at home are in corporate environments where outside software can't be installed, but PowerToys is available. I cannot express how much my work flow is hindered by the lack of this ability at work.

If you created a Patreon, GoFundMe, etc for this, I'll contribute and I'm sure others will too. If I can fund it on my own I will.

Windows is usable without this!

**Comment by joosniz** [comment]
I'm 2.5 years late to the discussion but FYI your link is broken, it's pointing to `https://github.com/RamonUnch/AltSnap/discussions/url` instead of https://github.com/microsoft/PowerToys/issues/269

I agree though. AltSnap has so many features but I'd be content if I had the basics at least on my work PC, dragging or scaling a window without hunting for the right pixel to click makes me much more productive, really should be a native Windows feature but a Power Toys component would be the next best thing.

**Comment by RamonUnch** [comment]
It was implemented in PowerToys (very basic version).
https://github.com/microsoft/PowerToys/pull/47024
